src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -61,6 +61,65 @@
 
   </Target>
 
+  <!--
```

Comment:
In theory, you can have multiple resource prefixes with IPhoneResourcePrefix, separated by a semi-colon:

```xml
<IPhoneResourcePrefix>Resources;Platforms/iOS/Resources</IPhoneResourcePrefix>
```

It looks like our code supports this (that's the intention at least), but I've never seen it done, and we don't have tests for this, so YMMV... but if it works, it would be a much simpler solution.

Reply:
Oh! Wow, let us do that quick and see :)

Reply:
Hmm, i also see this:
https://github.com/xamarin/xamarin-macios/blob/76769f17d50768e9c0ed77ad5b6f202753083e32/dotnet/targets/Microsoft.Sdk.DefaultItems.template.props#L62

Reply:
It looks like it built... I can also do something that Blazor does and just before the CollectBundleResources I update _ResourcePrefix to be 2 and then reset just after it runs? Not sure that is bad or worse.

Reply:
Hm yeah, using _ResourcePrefix in an Include like that won't work as expected with multiple resource prefixes.

You could try updating _ResourcePrefix just before CollectBundleResources to see if that works. I'd want to implement a proper fix though, probably make that Include work with multiple resource prefixes, so the idea would for it to be a temporary workaround.

Reply:
Doing that for now and opened an issue: https://github.com/xamarin/xamarin-macios/issues/20968